specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,9 +6,19 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
-The current preview release is 2022-02-01-preview
+The current preview release of Conversational LU and TextAnalytics is 2022-03-01-preview.
```

Comment:
For product names, I'd spell out what they are for customers, IMO:
```suggestion
The current preview release of Conversational Language Understanding and Text Analytics is 2022-03-01-preview.
```

Reply:
Fixed